### ISSUE TITLE
Awkwardly placed comma

### ISSUE THREAD

**Issue by teepeemm** [APEXCalculus/APEXCalculusV5#10]
https://github.com/APEXCalculus/APEXCalculusV5/blob/2ec7e64191dfabbf1497bc7a00d3e2989614303b/exercises/08_07_ex_36.tex#L1

This isn't technically a mistake, but `\ds\frac2y,` looks very similar to `\ds\frac2{y'}`.  I'm going to try `\ds\frac2y;` and change to semicolons for the adjacent problems.

**Comment by APEXCalculus** [comment]
Yes, that is tricky. I couldn't tell what you were referring to until I saw it typeset.
